CMakeLists.txt

```diff
@@ -80,6 +79,11 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/Test-Suite")
     include(CTest)
 endif()
 
+option(WITH_FSTBHC "Build SVF with FSTBHC enabled" OFF)
```

Comment:
This can also be removed

Reply:
Yes this is removed
```
mbarbar@alder ~/clone/svf-public % ag fstbhc
mbarbar@alder ~/clone/svf-public % ag tbhc
include/SVF-FE/DCHG.h
300:        // For TBHC this is conservative because the union type is lower in the DCHG
mbarbar@alder ~/clone/svf-public %
```